lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -25,6 +26,12 @@ define((require, exports, module) ->
   exports.LIST_STATE = LIST_STATE
   exports.EQUATION_TOKENTYPE = EQUATION_TOKENTYPE
   exports.LIST_TOKENTYPE = LIST_TOKENTYPE
+  exports.ENVIRONMENT_STATE = "environment"
```

Comment:
@khazhoyan this is not used anywhere, so there's no need for local `ENVIRONMENT_STATE`.